src/components/user.js

```diff
@@ -15,69 +25,136 @@ export default class User extends React.Component {
         const { info = {}, address = {}, company = {} } = this.props;
 
         this.state = {
-            currentUser: {
+            currentUser:         {
                 info,
                 address,
                 company
             },
-            showAddress: false,
-            showCompany: false,
-            wantEdit:    false
+            unmodifiedUser:      {
```

Comment:
we can init it as `null`. We don't need to have it untill editing process is started